enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -386,13 +380,12 @@ As a first step a similar high cardinality alert will be used as in [9] and pote
 
```

Comment:
@paulfantom I did not touch the "### High cardinality detection" part, when you get a chance do you mind updating it with the recent outcome of the spike you did in a new PR, thanks!